.github/workflows/main.yml

```diff
@@ -128,15 +128,61 @@ jobs:
 
   publish-flextesa-plugin-to-npm:
     needs:
-      - publish-ligo-plugin-to-npm
-      - publish-smartpy-plugin-to-npm
-      - publish-taquito-plugin-to-npm
+      - publish-sdk-to-npm
     uses: ecadlabs/taqueria/.github/workflows/npm-publish.yml@main
     with:
       dir: taqueria-plugin-flextesa
     secrets:
       NPM_TOKEN: ${{ secrets.NPMJS_PAT }}
 
+  run-tests:
+    needs: build-binaries
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./tests
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 16.x
+
+      - uses: denoland/setup-deno@v1
```

Comment:
I'll make it specific to use the latest one

Reply:
According to https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-versioned-actions `v1` pins it to the latest major version. So it's better to keep it as you originally wrote it so that we don't need to update it manually